docs/source/usage/picmi/intro.rst

```diff
@@ -11,6 +11,17 @@ Example
 
 Creates a directory ``generated_input``, where you can run ``pic-build`` and subsequently ``tbg``.
 
+.. note::
+
+   Please be aware that for running the python script, you need to setup a python environment that includes all
+   dependecies listed in ``$PICSRC/lib/python/picongpu/picmi/requirements.txt``. This can be done by e.g.
+   ``pip install -r requirements.txt``. You also need to setup your ``$PYTHONPATH`` to include PIConGPU's
+   python libraries ``export PYTHONPATH=$PICSRC/lib/python:$PYTHONPATH``. (This is commonly done by your PIConGPU environment,
+   see below.)
+   For ``pic-build`` and ``tbg`` you need to setup a working PIConGPU environment. This is for example documented
+   in `the Setup part of PIConGPU in 5 Minutes on Hemera <hemeraIn5min>`. 
+
+
```

Comment:
```suggestion
.. note::

Remember to install the PICMI python dependencies, see ``lib/python/picongpu/picmi/requirements.txt``, to your environment before using PICMI. This can be done by e.g. ``pip install -r requirements.txt`` (called from the PIConGPU source code directory).

In addition you must add the picongpu python modules to your PYTHONPATH, e.g. ``export PYTHONPATH=$PICSRC/lib/python:$PYTHONPATH`` and provide the environment variables,

- ``PICSRC`` ... path of picongpu source code directory
- ``PIC_EXAMPLES`` ... path to picongpu examples, usually ``$PICSRC/share/picongpu/examples`` 
- ``PIC_SYSTEM_TEMPLATE_PATH`` .. path to picongpu tpl file

For ``pic-build`` and ``tbg`` you need to setup a working PIConGPU environment. This is for example documented in `the Setup part of PIConGPU in 5 Minutes on Hemera <hemeraIn5min>`. 
```
In general this in formation should be moved to a separate installation documentation page and only referenced here. We suggest to do fix this in separate PR.